components/CandidateTestView/Tests/HelloWorld.tsx

```diff
@@ -0,0 +1,16 @@
+
+export const HelloWorldTest: CandidateTestModel = {
+    number: 2,
+    title: "Hello World!",
+    expectedOutput: "Hello, World!\n" + "Welcome to 10 Days of JavaScript!",
+    input: "Welcome to 10 Days of JavaScript!",
+    brief: "In this challenge, we review some basic concepts that will get you started with this series. Check out the tutorial to learn more about JavaScript's lexical structure.\n" + "A greeting function is provided for you in the editor. It has one parameter, parameter variable. Perform the following tasks to complete this challenge:\n\"Use console.log() to print Hello, World! on a new line in the console, which is also known as stdout or standard output. The code for this portion of the task is already provided in the editor." + "Use console.log() to print the contents of  (i.e., the argument passed to main).\n" + "You've got this!",
+    example: "We printed two lines of output:\nWe printed the literal string Hello, World! using the code provided in the editor.\nThe value of parameter variable passed to our main function in this Sample Case was Welcome to 10 Days of JavaScript!. We then passed our variable to console.log, which printed the contents of parameter variable.",
+    executionTimeLimit: DefaultExecutionTimeLimit,
+    inputType: "string",
+    inputDescription: "A single line of text containing one or more space-separated words.",
+    outputType: "string",
+    outputDescription: "Print the following two lines of output:\n" + "On the first line, print Hello, World! (this is provided for you in the editor).\n" + "On the second line, print the contents of parameter variable (i.e., the argument passed to main).\n",
+    modelAnswer: "'use strict';\n" + "\n" + "process.stdin.resume();\n" + "process.stdin.setEncoding('utf-8');\n" + "\n" + "let inputString = '';\n" + "let currentLine = 0;\n" + "\n" + "process.stdin.on('data', inputStdin => {\n" + "    inputString += inputStdin;\n" + "});\n" + "\n" + "process.stdin.on('end', _ => {\n" + "    inputString = inputString.trim().split('\\n').map(string => {\n" + "        return string.trim();\n" + "    });\n" + "    \n" + "    main();    \n" + "});\n" + "\n" + "function readLine() {\n" + "    return inputString[currentLine++];\n" + "}\n" + "\n" + "/**\n" + "*   A line of code that prints \"Hello, World!\" on a new line is provided in the editor. \n" + "*   Write a second line of code that prints the contents of 'parameterVariable' on a new line.\n" + "*\n" + "*\tParameter:\n" + "*   parameterVariable - A string of text.\n" + "**/\n" + "function greeting(parameterVariable) {\n" + "    // This line prints 'Hello, World!' to the console:\n" + "    console.log('Hello, World!');\n" + "\n" + "    // Write a line of code that prints parameterVariable to stdout using console.log:\n" + "    \n" + "}\n" + "\n" + "\n" + "function main() {\n" + "    const parameterVariable = readLine();\n" + "    \n" + "    greeting(parameterVariable);\n" + "}",
```

Comment:
we don't seem to render this anywhere - can we remove it?

Reply:
This is one of the tests. As we are not connecting to DB, I have been storing alternate tests in this way. So if you select index 3 from the test list this test will be rendered

Reply:
yep - but we don't ever use the `modelAnswer` right? we definitely don't want to include that in things we send to the candidate.

We surely don't want to have a `modelAnswer` in the candidates repo?

Reply:
Ah, I think modelAnswer might not be the best description. Its actually the starter code that will appear in the editor - renamer to testStarterCode?